src/widgets/dock_area/mod.rs

```diff
@@ -501,14 +509,14 @@ impl<'tree, Tab> DockArea<'tree, Tab> {
                 tabs_ui.output_mut(|o| o.cursor_icon = CursorIcon::Grabbing);
             }
 
-            let (is_active, label, tab_style) = {
+            let (is_active, label, tab_styles) = {
```

Comment:
Why the plural? To me, it implies that `tab_styles` is a collection of styles, even though it's clearly a single style struct. Granted, `TabStyle` contains a few other style structs, but together they all describe a single style.

Reply:
No reason, i had it plural before but then changed it. I changed other occurrences of this as well.